definitions/features/service.rb

```diff
@@ -61,7 +61,7 @@ def use_system_service(action, options, spinner)
   def run_action_on_services(action, options, spinner)
     status = 0
     failed_services = []
-    filtered_services(options).each_value do |group|
+    filtered_services(options, action).each_value do |group|
```

Comment:
As per the rubocop optional arguments should be the last and so `filtered_services` has `action` as last argument. But then for `filter_services` its not optional hence I kept it by alphanumerically sorted?

Reply:
I would suggest to always add new arguments to methods as the last one. That's the safest because you don't need to re-order every call to it. In this case it also gives you a more consistent API: filtered_services is (options, action) and filter_services would be (service_list, options, action).

Now in this case it's not as critical since it's a required argument, but if it was an optional argument then it's very easy to introduce bugs because no exception would be raised. That's why I'll leave it up to you if you want to make it `def filter_services(options, service_list, action)` or not. However, I do hope you understand why I made the suggestion.

Reply:
Oh, I now see my bad, but I meant `def filter_services(service_list, options, action)`. My apologies!